File: CardIndex/DAL/Entities/CardAssessment.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace DAL.Entities
{
    public class CardAssessment
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }

        public int Rate { get; set; }
        public string Comment { get; set; }

        public DateTime Date { get; set; }
        [ForeignKey(nameof(Card))]
        public int CardId { get; set; }
        public string UserId { get; set; }

        public virtual Card Card { get; set; }
    }
}

File: CardIndex/CardIndex/ClientApp/src/app/models/card-asessment-models/card-assessment.ts
export class CardAssessment{
    id: number
    rate: number
    comment: string
    cardTitle: string
    userLogin: string
    date: Date

    constructor(
        id?: number,
        rate?: number,
        comment?: string,
        cardTitle?: string,
        userLogin?: string,
        date?: Date,
    )
        {
            this.id = id;
            this.rate = rate;
            this.comment = comment;
            this.cardTitle = cardTitle;
            this.userLogin = userLogin;
            this.date = date;
        }
}
